build.gradle

```diff
@@ -43,14 +43,14 @@ dependencies {
     shadow('com.fasterxml.jackson.core:jackson-databind:2.15.0-rc1')
 
     // Thank you georgeagostino for fixing my garbage
-//    if (getProject().hasProperty("altoclef.development")) {
-    // Must run build from baritone-plus once
-    modImplementation 'baritone-api-fabric:baritone-unoptimized-fabric-1.19.4-beta1'
-    include "baritone-api-fabric:baritone-unoptimized-fabric-1.19.4-beta1"
-    //} else {
-    //    modImplementation "cabaletta:baritone-unoptimized-fabric:1.18.2"
-    //    include "cabaletta:baritone-unoptimized-fabric:1.18.2"
-    //}
+    if (getProject().hasProperty("altoclef.development")) {
+        // Must run build from baritone-plus once
+        modImplementation 'baritone-api-fabric:baritone-unoptimized-fabric-1.19.4-beta1'
+        include "baritone-api-fabric:baritone-unoptimized-fabric-1.19.4-beta1"
+    } else {
+        modImplementation "cabaletta:baritone-unoptimized-fabric:1.19.4-beta1"
+        include "cabaletta:baritone-unoptimized-fabric:1.19.4-beta1"
+    }
```

Comment:
I'm not sure you can just grab Cabaletta like this. Alto uses its own baritone and as such, needs the unoptimized jar. That's probably why the gradle is failing on the workflow.

Reply:
More specifically, it won't be able to actually use the cabaletta baritone because our baritone actually references altoclef

Reply:
It's for my maven.